sql/core/src/test/resources/sql-tests/inputs/datetime-formatting-exception.sql

```diff
@@ -0,0 +1,2 @@
+--SET spark.sql.legacy.timeParserPolicy=EXCEPTION
+--IMPORT datetime-formatting.sql
```

Comment:
I don't see any exception in the result golden file. Do we really need to test it again with EXCEPTION policy?

Reply:
I'm puzzling about this as well.
I cannot find where we explicitly tested CORRECTED before, although EXCEPTION was the default.
I expected copy the existing tests, label them EXCEPTION and it should behave as before.
And the existing files, with the new default should have no/fewer exceptions. This is not what I observe...

Reply:
OK, I have a better understanding of what CORRECTED actually does.
Removed all the extra files.
I also found that there was SQ:ApiConf.scala needed to be fixed. That flushed out the expected changes for CORRECTED